migrator/zenodo_rdm_migrator/actions/drafts.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Migrator is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio RDM migration drafts row load module."""
+
+
+from invenio_rdm_migrator.actions import TransformAction
+from invenio_rdm_migrator.load.postgresql.transactions.operations import OperationType
+from invenio_rdm_migrator.streams.actions import RDMDraftCreateAction
+from invenio_rdm_migrator.transform import IdentityTransform
+
+from ..transform.records import ZenodoRecordTransform
+
+
+class ZenodoDraftCreateAction(TransformAction):
```

Comment:
note for reviewers:

It assumes to things:
- The data returned by the extractor has already been "filtered" (e.g. removed `schema`, set `payload` content as root`)
- Some types have been changes (e.g. `op` should be an instance of `OperationType`)